databases/week1/database.sql

```diff
@@ -0,0 +1,78 @@
+/*1) Find out how many tasks are in the task table */
+select count(*) 
```

Comment:
```suggestion
select count(0) 
```
You can also use the count(0) instead of count(*).Its give a same result but execute bit faster.

Reply:
Thanks for reviewing my hw... :)